### DISCUSSION TITLE
EV3 MicroPython v3.0.0-rc.1 image

### DISCUSSION THREAD

**Discussion by dvirabyan** [pybricks/support#370] [Q&A] [answered]
Hi!!

I was wondering if there is an image for EV3 bricks for the latest [Pybricks 3.0.0-rc.1](https://docs.pybricks.com/en/latest/parameters/color.html).

The image from [lego education website](https://education.lego.com/en-us/product-resources/mindstorms-ev3/teacher-resources/python-for-ev3) is v2.0.0.

So, is there a 3.0.0 version for EV3? There are new "Color" attributes I really want to use!

**Comment by dlech** [comment, marked as answer]
Hi. We haven't been focused on the ev3dev build lately but you can always grab the latest builds [here](https://github.com/pybricks/pybricks-micropython/actions/workflows/build.yml?query=is%3Asuccess). Click on the first item in the list (the most recent build) and then scroll down to *Artifacts* and click on *pybricks-micropython*. This will download a zip with a `pybricks-micropython` executable for ev3dev-stretch.

You can put this executable in `/usr/local/bin/` on the EV3 to override the 2.0 version (and delete it to go back to v2.0). But we haven't tested it in a while, so who knows what will happen. :smile: 

Shortcut for today's most recent build: [link](https://github.com/pybricks/pybricks-micropython/suites/3003726393/artifacts/67940441).

**Reply by dvirabyan** [reply]
Thanks! Will give it a try.

**Comment by DJshelfmushroom** [comment]
@dlech sorry to bring up an old thread but is ev3dev no longer supported? in the nightly builds i don't see any unexpired pybricks-micropython builds.

**Reply by dlech** [reply]
Correct. We are working on a bare-metal EV3 firmware to replace it. There were some issues with motor driver control code in Pybricks v3.x that didn't work well on Linux, so we decided to go this route instead.

**Reply by DJshelfmushroom** [reply]
Thanks! Would it be possible for you to provide an old build in the meantime while I wait for that? I'm trying to upgrade (or reinstall i suppose) our ev3's with buster right now, so any advice would be appreciated.

**Reply by dlech** [reply]
Pybricks 2.x ships with ev3dev, so you can continue to use that.
